webapp/app.py

```diff
@@ -41,13 +30,25 @@
 app.static_folder = "../static"
 app.url_map.strict_slashes = False
 
+discourse_api = DiscourseAPI(
+    base_url="https://forum.snapcraft.io/",
+    frontpage_id=3781,  # The "Snap Documentation" topic
+    category_id=15,  # The "doc" category
+)
+DiscourseDocs().init_app(
```

Comment:
If app is defined in the constructor it calls the function `init_app`

```python
DiscourseDocs(
    app=app,
    model=discourse_api,
    url_prefix="/",
    document_template="document.html",
)
```

Reply:
Yes I was wondering about that. I followed your QA steps too faithfully :p

Reply:
We agreed not to change this